commands to create job and cronjob (#24701)
kubernetes/website: content/en/docs/reference/kubectl/cheatsheet.md

`.yml`, and `.json` can be used.

```bash
kubectl apply -f ./my-manifest.yaml                                                               # create resource(s)
kubectl apply -f ./my1.yaml -f ./my2.yaml                                                         # create from multiple files
kubectl apply -f ./dir                                                                            # create resource(s) in all manifest files in dir
kubectl apply -f https://git.io/vPieo                                                             # create resource(s) from url
kubectl create deployment nginx --image=nginx                                                     # start a single instance of nginx
kubectl create job hello --image=busybox -- echo "Hello World"                                    # create a job which prints "hello world"

> kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"       # create a cron job which prints "hello world" for every minute

Reviewer: Thanks for adding this hints. I'd suggest we put the new lines this way:

```
kubectl create job hello \                     # create a job which prints "hello world"
   --image=busybox -- echo "Hello World"                                    
```

The intent is to avoid horizontal scrolling of the code segment considering that readers may have a screen not so big.

Author: @tengqm - Good feedback. I have made the change. Please review

kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
kubectl apply -f https://git.io/vPieo          # create resource(s) from url
kubectl create deployment nginx --image=nginx  # start a single instance of nginx
kubectl create job hello                       # create a job which prints "hello world"
--image=busybox -- echo "Hello World"                                    
kubectl create cronjob hello --image=busybox   # create a cron job which prints "hello world" for every minute

> --schedule="*/1 * * * *" -- echo "Hello World"       

Reviewer: nits:  please consider indent line 93 and 96 by 2+ spaces so users can easily see those lines are not standalone commands;

> --schedule="*/1 * * * *" -- echo "Hello World"       

Reviewer: @sftim does have some valid concerns. So maybe we should use the format here:

https://github.com/kubernetes/website/pull/24701/files#diff-9b5338cfe4b946f1b17ff38106ad8410dc6687e17e8327d3fc4864a5ee8a48c5R71-R73

Reviewer: about line 96 and 97, you can either do 

```
kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"
```

or 

```
kubectl create cronjob hello --image=busybox \
  --schedule="*/1 * * * *" -- echo "Hello World"
```

The current format is confusing. It looks like two commands.

kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
kubectl apply -f https://git.io/vPieo          # create resource(s) from url
kubectl create deployment nginx --image=nginx  # start a single instance of nginx

> # create a job which prints "hello world"

Reviewer 2: ```suggestion
# create a Job which prints "Hello World"
```

Author: @sfitm - done this change

kubectl create job hello --image=busybox -- echo "Hello World" 

> # create a cronjob which prints "hello world" for every minute

Reviewer 2: ```suggestion
# create a CronJon that prints "Hello World" every minute
```

Author: @sftim - done this change

Author: /hold

Reviewer 2: > /hold

?
(I'm not sure why to hold)
